File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    for val in sizes:
        val.sort()
    
    maxW = -1
    maxH = -1
    for val in sizes:
        if val[0] > maxW:
            maxW = val[0]
        if val[1] > maxH:
            maxH = val[1]
            
    return maxW * maxH

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let temp = sizes;
    for (let i = 0; i < sizes.length; i++) {
        temp[i].sort((a, b) => a - b);
    }
    
    let maxW = -1;
    let maxH = -1;
    for (let i = 0; i < sizes.length; i++) {
        if (temp[i][0] > maxW) {
            maxW = temp[i][0];
        }
        if (temp[i][1] > maxH) {
            maxH = temp[i][1];
        }
    }
    
    return maxW * maxH;
}